src/parser.rs

```diff
@@ -480,6 +480,7 @@ impl Parser {
                             span_end,
                         );
                     }
+                    // TODO: Are we going to support method calls with dot syntax?
```

Comment:
you mean like `foo.bar 1 2 3`?

I don't believe we support that. This might just be some copy/paste functionality from the other project.

Reply:
OK, I removed that